trunk/SUAVE/Optimization/Package_Setups/particle_swarm_optimization.py

```diff
@@ -0,0 +1,191 @@
+## @ingroup Optimization-Package_Setups
+# particle_swarm_optimization.py
+# 
+# Created:  Sep. 2019, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import numpy as np
+import scipy as sp
+ 
+## @ingroup Optimization-Package_Setups
+def particle_swarm_optimization(func, lb, ub, ieqcons=[], f_ieqcons=None, args=(), kwargs={}, 
+        swarmsize=100, omega=0.5, phip=0.5, phig=0.5, maxiter=100, 
+        minstep=1e-8, minfunc=1e-8, debug=False):
+    """
+    This function perform a particle swarm optimization (PSO)
```

Comment:
What's the source for this method? Make sure everything from the standard SUAVE docstring is included.

Reply:
changed to standard docstring format